docs/aboutus.rst

```diff
@@ -475,22 +475,22 @@ competitive and fast (from 10x to 100x) by rewriting the core from scratch.
 Jurjen N.E. Bos has contributed pretty printing and other patches.
 Fredrik Johansson has wrote mpmath and contributed a lot of patches.
 
-SymPy has participated in every GSoC since 2007.  Moderate amount
+SymPy has participated in every GSoC since 2007. Moderate amount
```

Comment:
Two spaces after the sentence - is not a grammar error.  This is a "Emacs thing" and more or less - convention for open source world, part of GNU coding standard.